frontend/app/src/components/StreamlitDialog/SettingsDialog.test.tsx

```diff
@@ -16,7 +16,7 @@
 
 import React from "react"
 
-import userEvent from "@testing-library/user-event"
+import { default as userEvent } from "@testing-library/user-event"
```

Comment:
what's up with this?

Reply:
So this is an eslint rule that warns you if there's an issue, which fails CI. I figure it was easier to migrate it to the setup to comply with the eslint rule than figure out how to disable it.

Reply:
the part that confuses me is why 
`import userEvent from "@testing-library/user-event"`
was getting flagged by eslint in the first place

while this syntax
`import { default as userEvent } from "@testing-library/user-event"`
makes sense, it seems... redundant?

Reply:
fwiw, it looks like you can sidestep it all together and just import it via a named import 🤷‍♂️ 

https://github.com/testing-library/user-event/blob/main/src/index.ts

Reply:
Oh nice! I'll update all places. There are other places where this syntax is used. We should have a conversation about the eslint rules with our formal recommendation.